kbase-extension/static/kbase/js/common/jobCommChannel.js

```diff
@@ -105,26 +106,6 @@ define([
             // Tells kernel to stop including a job in the regular job status updates
             STOP_UPDATE,
         },
-        BACKEND_REQUESTS = {
-            CANCEL,
-            CELL_JOB_STATUS,
-            INFO,
-            LOGS,
-            RETRY,
-            START_UPDATE,
-            STATUS,
-            STOP_UPDATE,
-        },
-        BACKEND_RESPONSES = {
-            CELL_JOB_STATUS,
-            ERROR,
-            INFO,
-            LOGS,
-            RETRY,
-            RUN_STATUS,
-            STATUS,
-            STATUS_ALL,
-        },
```

Comment:
the frontend and backend now use the same names for the message types, so this can all be removed